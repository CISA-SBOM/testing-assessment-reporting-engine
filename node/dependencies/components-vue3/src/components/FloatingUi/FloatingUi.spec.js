import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from "./FloatingUi.vue";

describe("FloatingUi.vue", () => {
  it("is a Vue instance", () => {
    const wrapper = shallowMount(Component);
    expect(wrapper.vm).toBeTruthy();
  });

  it("matches snapshot with no props assigned", async () => {
    const props = {};
    const wrapper = shallowMount(Component, { props });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
