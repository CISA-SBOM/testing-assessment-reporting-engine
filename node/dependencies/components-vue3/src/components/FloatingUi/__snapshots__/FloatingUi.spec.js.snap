// Vitest Snapshot v1

exports[`FloatingUi.vue > matches snapshot with no props assigned 1`] = `
"<div class=\\"inline-block\\">
  <div class=\\"inline-block w-full\\"></div>
  <!---->
</div>"
`;
